scripts/SREP/infra-node-resize/script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Infra-node-resize
+
+set -x
+set -e
+set -o nounset
+set -o pipefail
+
+# find hive_shard ID
+CLUSTER_ID=ocm list clusters --parameter search="external_id is '${1}'" --columns id --padding 20 --no-headers
+HIVE_ID=$(ocm get /api/clusters_mgmt/v1/clusters/${CLUSTER_ID}/provision_shard | jq -r '.hive_config.server' | cut -d '.' -f 2)
```

Comment:
This can be changed to a better name as we'll have the name of the hive cluster and no such thing as 'HIVE ID'